en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -47,7 +47,13 @@ Adding third-party JARs to the Micro Integrator can provide the following benefi
 
 To add third-party libraries to the Micro Integrator, you can add a new directory named `libs` to the root of the Micro Integrator project. When you deploy the component, all the JARs in the `libs` directory in the project path will be included in the Micro Integrator runtime.
 
-![Libs Directory](/assets/img/ipaas/integration-studio/libs_dir.png){.cInlineImage-half}
+![Libs Directory](../../assets/img/ipaas/integration-studio/libs_dir.png){.cInlineImage-half}
+
+## Scan Third-party libraries to identify security vulnerablities
+
+Conducting security vulnerability scans is crucial for identifying potential weaknesses in an application that could be exploited by attackers. This proactive approach enables organizations to detect and mitigate security risks before they are exploited, thereby preventing data breaches, system compromises, and other security incidents.
+
+During deployment, Choreo performs a Trivy scan to detect critical vulnerabilities in third-party libraries added to the integration component. If the scan detects any critical vulnerabilities, it halts the deployment process. The deployment panel shows the Trivy scan status and any security failures in the Library (Trivy) vulnerable scan step, which you can access by clicking on it. After fixing the vulnerability, you can redeploy the component.
```

Comment:
Shall we change panel to pane to adhere with the Google Style Guide.

Reply:
done